package.json

```diff
@@ -295,6 +295,11 @@
         "title": "Show package documentation",
         "category": "LaTeX Workshop"
       },
+      {
+        "command": "latex-workshop.texdocUsepackages",
+        "title": "Show package documentation actually used",
+        "category": "LaTeX Workshop"
+      },
```

Comment:
Good idea to add an extra function to access the documentation of the included packages. This solves #1692.